client/components/map.jsx

```diff
@@ -89,6 +89,10 @@ export default function Map(props) {
     setModal(true);
   };
 
+  const addSoundscapeMarker = marker => {
+    soundscapeMarkers.push(marker);
+  };
```

Comment:
Rather than modify the existing array, we want to update our component's state to include the new marker.

```js
  const addSoundscapeMarker = marker => {
    setSoundscapeMarkers(markers => markers.concat(marker));
  };
```